src/GitVersionCore.Tests/IntegrationTests/MainlineDevelopmentMode.cs

```diff
@@ -530,6 +530,69 @@ public void VerifyIncrementConfigIsHonoured()
         }
     }
 
+    public class MainlineDevelopmentModeWithBadConfig : TestBase
```

Comment:
Could you please place this class in its own file?

Reply:
Sure. Let me know if there is anything else.